doc/needles.txt

```diff
@@ -1,90 +0,0 @@
-Some packages are required to run crop.py. So for the first time do:
```

Comment:
The documentation *is* in the openqa repo - which part is in which repository is of no relevance to me. Perhaps we should add a pointer to the README though